proxy/src/registry.rs

```diff
@@ -486,16 +480,15 @@ impl Client for Registry {
             }))
     }
 
-    async fn list_org_projects(&self, id: String) -> Result<Vec<Project>, error::Error> {
-        let org_id = protocol::Id::try_from(id.clone())?;
-        let project_ids = self.client.list_projects().await?.into_iter();
+    async fn list_org_projects(&self, org_id: Id) -> Result<Vec<Project>, error::Error> {
+        let ids = self.client.list_projects().await?;
         let mut projects = Vec::new();
-        for project_id in project_ids {
-            if project_id.1 == org_id {
+        for id in &ids {
+            if id.1 == org_id.clone().0 {
```

Comment:
Would be good to have a getter here instead of the `.1`, eg. `id.domain() == org_id.as_str()` would be more readable

Reply:
This would be feedback for the radice-registry client implementation.

Reply:
Oh I see :(